src/main/java/org/broadinstitute/hellbender/utils/dataflow/DataflowUtils.java

```diff
@@ -67,6 +73,50 @@ public void processElement(ProcessContext c) {
                 .apply(ParDo.of(new LoadReadsFromFileFn(intervals)));
     }
 
+    /**
+     * Ingest BAM files from a Hadoop file system and load them into a
+     * <code>PCollection<Read></code>.
+     * @param pipeline a configured Pipeline
+     * @param intervals intervals to select reads from
+     * @param path Hadoop file or directory path to read BAM files from
+     * @return a <code>PCollection<Read></code> with all the reads that overlap the
+     * given intervals in the BAM files
+     */
+    public static PCollection<Read> getReadsFromHadoopBams(final Pipeline pipeline, final List<SimpleInterval> intervals, final String path) {
```

Comment:
We need to have tests for this.  I'm not totally sure how we should go about doing that because we don't have a hadoop filesystem set up in travis. 

 Is there an easy way to serve files as hdfs to test things?